File: 4-hashes_and_dicts.py
# Python dict 多數用 string 做 key，攞值時要用相同 key。
user = {
    "name": "Ada",
    "role": "admin",
    "active": True,
}

print(user["name"])
print(user["role"])
print(user["active"])

user["score"] = 95

for key, value in user.items():
    print(f"{key}: {value}")

File: 4-hashes_and_dicts.rb
# Ruby hash 好常用 symbol 做 key，例如 :name、:role。
user = {
  name: 'Ada',
  role: 'admin',
  active: true
}

puts user[:name]
puts user[:role]
puts user[:active]

user[:score] = 95

user.each do |key, value|
  puts "#{key}: #{value}"
end
